appveyor.yml

```diff
@@ -43,9 +43,8 @@ install:
   - ps: .\dev\appveyor-install-dependencies.ps1
   # Required package for R unit tests
   - cmd: R -e "install.packages(c('knitr', 'rmarkdown', 'e1071', 'survival'), repos='https://cloud.r-project.org/')"
```

Comment:
Would it be better to restore this line from #26041 to install arrow?
```
cmd: R -e "install.packages(c('knitr', 'rmarkdown', 'e1071', 'survival', 'arrow'), repos='https://cloud.r-project.org/')"
```

Reply:
Thanks. Yes. That will be better.